diff --git a/app/models/article.rb b/app/models/article.rb
index c39337e..5366b27 100644
--- a/app/models/article.rb
+++ b/app/models/article.rb
@@ -25,8 +25,11 @@ class Article < ActiveRecord::Base
     !!(published_at && published_at <= Time.current)
   end
 
+  # Monthly archive of published articles sorted descendingly
   def self.published_by_month
-    self.published.group_by { |article| article.published_at.beginning_of_month }.sort
+    self.published.group_by do |article|
+      article.published_at.beginning_of_month
+    end.sort.reverse!
   end
 
   include Rails.application.routes.url_helpers